README.md

```diff
@@ -55,6 +63,14 @@ Worldwide Holidays and Observances RESTful API
 - [Calendarific API](https://calendarific.com/api-documentation)
 - [API Documentation](https://calendarific.com/api-documentation)
 
+## Set Up API Key
+
+``
+
+1. To create .env.local file on the root path
+2. Inside file, declare a variable and give your API Key like this: NEXT_PUBLIC_API_KEY=Your API Key
```

Comment:
todo: 
- This should be written before the `yarn dev` step, because it's needed to run the project. 
- It's missing a `.env.local.example` to be easier to copy.